spec/models/subscription_spec.rb

```diff
@@ -114,6 +114,38 @@
     end
   end
 
+  describe ".unavailable_for_canceling?" do
```

Comment:
Looks like it's an instance method, so maybe `#unavailable_for_canceling` would be better since it's the common notation for instance methods, use `.method_name` for static/class methods or model scopes 😄